Weather App/viewModel/WeatherForCityViewModel.swift

```diff
@@ -4,25 +4,24 @@ import Keys
 
 protocol WeatherForCityViewModel {
     var weathers: [WeatherForCity] { get }
+    var weatherForLocation: WeatherForCity? { get }
     func getWeatherForCity(cityName: String) async
     func getWeatherForCoordinate(coordinate: CLLocationCoordinate2D) async
 }
 
 class WeatherForCityViewModelImpl: WeatherForCityViewModel {
     var weathers: [WeatherForCity] = []
+    var weatherForLocation: WeatherForCity?
```

Comment:
CityWeather will be better, what do you think?

Reply:
👍